src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -3050,6 +3069,50 @@ private void onAlteration(final ThreadPoolExecutor executor, final String parent
         return observer;
     }
 
+    /**
+     * Given the loose app file and the source directory path, return a list of
+     * files that are specified in the loose app file and are in the source
+     * directory and should be omitted from watching.
+     * 
+     * @param looseAppFile Loose Application configuration file
+     * @param srcDirectoryPath the source directory path
+     * @return a list of files that should be omitted from watching as they are on
+     *         the source directory path and exist in the loose app config file
+     */
+    protected Collection<File> getOmitFilesList(File looseAppFile, String srcDirectoryPath) {
+        Collection<File> omitFiles = new ArrayList<File>();
+        try {
+            if (looseAppFile != null && looseAppFile.exists()) {
+                DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+                DocumentBuilder db = dbf.newDocumentBuilder();
+                Document document = db.parse(looseAppFile);
+                NodeList archiveList = document.getElementsByTagName("archive");
+                for (int i = 0; i < archiveList.getLength(); i++) {
+                    NodeList ar = archiveList.item(i).getChildNodes();
+                    for (int j = 0; j < ar.getLength(); j++) {
+                        Node node = ar.item(j);
+                        if (node.getNodeName().equals("dir") || node.getNodeName().equals("file")) {
+                            String srcOnDiskNodeText = node.getAttributes().getNamedItem("sourceOnDisk").getTextContent();
+                            if (container) {
+                                srcOnDiskNodeText = srcOnDiskNodeText.replace("${" + DEVMODE_PROJECT_ROOT + "}",
+                                        getLooseAppProjectRoot(projectDirectory, multiModuleProjectDirectory)
+                                                .getCanonicalPath());
+                            }
+                            File srcOnDiskFile = new File(srcOnDiskNodeText);
+                            if (srcOnDiskFile.getCanonicalPath().startsWith(srcDirectoryPath)) {
+                                omitFiles.add(srcOnDiskFile);
+                            }
+                        }
+                    }
+                }
+            }
+        } catch (ParserConfigurationException | SAXException | IOException e) {
+            debug("Unable to read loose application configuration file: " + looseAppFile.toString());
```

Comment:
Should this log as error so that any underlying issues would be noticed?  Although there is not much the user can do in this case.

Reply:
I've logged this as an error so at least users will be aware